pymc4/_template_contexts.py

```diff
@@ -58,10 +60,13 @@ class InferenceContext(BaseContext):
 
     def __init__(self, tensors, expected_vars):
         self.vars = []
+        self._names = [var.name for var in expected_vars]
         self._tensors = {var.name: tensor for var, tensor in zip(expected_vars, tensors)}
+        return
 
     def add_variable(self, rv):
         self.vars.append(rv)
+        return
```

Comment:
Is this black?

Reply:
No its me adding it so I have a place to activate the inline debugger after all the attribute have been instantiated.